ui/raidboss/data/04-sb/trial/shinryu.js

```diff
@@ -0,0 +1,549 @@
+      infoText: {
+        en: 'Out of middle',
+        de: 'Raus aus der Mitte',
+        cn: '离开中间',
+        ko: '중앙 피하기',
+      },
+    },
+    {
+      id: 'Shinryu Normal Hellfire',
+      netRegex: NetRegexes.startsUsing({ id: '1FAB', source: 'Shinryu', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '1FAB', source: 'Shinryu', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '1FAB', source: 'Shinryu', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '1FAB', source: '神龍', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1FAB', source: '神龙', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '1FAB', source: '신룡', capture: false }),
+      durationSeconds: 7,
+      alertText: {
+        en: 'Get in water',
+        de: 'In\'s Wasser',
+        cn: '进水圈',
+        ko: '물 장판에 들어가기',
+      },
+      tts: {
+        en: 'water',
+        de: 'wasser',
+        cn: '进水圈',
+        ko: '물 장판',
+      },
+    },
+    {
+      id: 'Shinryu Normal Hypernova',
+      netRegex: NetRegexes.startsUsing({ id: ['1F99', '1F9A'], source: 'Right Wing', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: ['1F99', '1F9A'], source: 'Recht(?:e|er|es|en) Schwinge', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: ['1F99', '1F9A'], source: 'Aile Droite', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: ['1F99', '1F9A'], source: 'ライトウィング', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: ['1F99', '1F9A'], source: '右翼', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: ['1F99', '1F9A'], source: '오른쪽 날개', capture: false }),
+      durationSeconds: 7,
+      alertText: {
+        en: 'Stack in water',
+        de: 'In Wasser stacken',
+        cn: '在水圈攻击',
+        ko: '물 장판에 모이기',
+      },
+      tts: {
+        en: 'water',
+        de: 'Wasser',
+        cn: '水圈',
+        ko: '물 장판',
+      },
+    },
+    {
+      id: 'Shinryu Normal Judgement Bolt',
+      netRegex: NetRegexes.startsUsing({ id: '1FAC', source: 'Shinryu', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '1FAC', source: 'Shinryu', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '1FAC', source: 'Shinryu', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '1FAC', source: '神龍', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '1FAC', source: '神龙', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '1FAC', source: '신룡', capture: false }),
+      durationSeconds: 7,
+      alertText: {
+        en: 'out of water',
+        de: 'Raus aus dem Wasser',
+        cn: '离开水圈',
+        ko: '물 장판 밖으로',
+      },
+    },
+    {
+      id: 'Shinryu Normal Levinbolt',
+      netRegex: NetRegexes.startsUsing({ id: '1F9B', source: 'Right Wing', target: 'Right Wing', capture: false }),
```

Comment:
What does the target specifier get you here? Mostly curious because nothing else uses this.

Reply:
Nothing, that's a copypasta leftover from Extreme. No idea why it was used there, honestly.